src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/validation/ValidationService.kt

```diff
@@ -1031,6 +1053,55 @@ class ValidationService(
     return emptyList()
   }
 
+  private fun validateToreraExempt(sentenceAndOffences: List<SentenceAndOffenceWithReleaseArrangements>): List<ValidationMessage> {
```

Comment:
With the class now over 1000 lines, I could move the code into a separate class or function file. It depends how we want to scale.